crates/weaver_resolver/src/lib.rs

```diff
@@ -145,6 +146,17 @@ pub enum Error {
     /// A container for multiple errors.
     #[error("{:?}", format_errors(.0))]
     CompoundError(#[related] Vec<Error>),
+
+    /// An error indicating that the Body references are not resolved.
+    #[error("The following body reference is not resolved.\nBody reference: {group_id}\nProvenance: {provenance}\nError: {error}")]
+    UnresolvedBody {
```

Comment:
Not sure this error is required anymore.

Reply:
Yep, it no longer seems to be referenced -- removing and testing locally